<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="LoginLable">
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="LoginText">
     <property name="toolTip">
      <string>Enter login here.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="PasswordLable">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="PasswordText">
     <property name="toolTip">
      <string>Enter password here.</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="ServerLable">
     <property name="text">
      <string>Server:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="ServerText">
     <property name="toolTip">
      <string>Adress of database server.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ServerText</tabstop>
  <tabstop>LoginText</tabstop>
  <tabstop>PasswordText</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LoginDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoginText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ServerText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PasswordText</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginDialog</receiver>
   <slot>onTextChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChanged()</slot>
 </slots>
</ui>
